src/ignitus-WhatWeDo/Components/CardLayout.js

```diff
@@ -21,15 +21,15 @@ const CardLayout = () => {
     },
     {
       image: Expert,
-      title: 'Experts Community',
+      title: 'Community of Experts',
       description:
-        'Guidance from an unparalleled network of professors,experts and industry professionals supervising internal projects with selected students',
+        'Guidance from an unparalleled network of professors, experts and industry professionals supervising internal projects with selected students.',
     },
   ];
-
-  const html = [...data].map((x, key) => (
+  // I would suggest not giving these cards a hover effect because it implies that they are links
+  const html = [...data].map(x => (
     <Card
-      key={key}
+      key={x.title}
```

Comment:
good ! using index as key as an anti-pattern but instead using the title or source (in the files above) I would suggest to generate a unique key like: `var shortid = require('shortid'); key={shortid.generate()}`

Reply:
@meyfleur I agree on the same good suggestion 👍

Reply:
I would agree with each of you if we intended to generate hundreds or more cards, but seeing as I don't think the card count will reach those numbers, I think generating a unique key for each would be a bit much. The same logic could be achieved by simply doing something like `key={Math.random().toFixed(4)}` or `key={`${x.title}${Math.random()}`}`